yaksh/demo_templates/yaml_question_template

```diff
@@ -0,0 +1,462 @@
+---
```

Comment:
This template is a bit backwards. I'd like the description to be first and the testcase last, perhaps we should dump the dict in sorted order, maybe use an ordered dict?